.github/workflows/ci.yml

```diff
@@ -30,10 +32,14 @@ jobs:
         runner: [macos-13, macos-15, ubuntu-22.04, windows-2025]
     runs-on: ${{ matrix.runner }}
     steps:
-    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
-    - uses: cgrindel/gha_set_up_bazel@v1
+    - name: Checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+    - name: Setup Bazel
+      uses: bazel-contrib/setup-bazel@4fd964a13a440a8aeb0be47350db2fc640f19ca8 # 0.15.0
```

Comment:
There was a clash on the cache-keys with the workflow that runs the integration tests, so I switched to another action to avoid this and be able to save the cache on both cases.